src/standalone/Bucket/Bucket.jl

```diff
@@ -156,11 +156,11 @@ should be used.
 function BulkAlbedoTemporal{FT}(
     regrid_dirpath::String,
     date_ref::Union{DateTime, DateTimeNoLeap},
-    t_start::FT,
+    t_start,
     space::ClimaCore.Spaces.AbstractSpace;
     infile_path = Bucket.cesm2_albedo_dataset_path(),
     varname = "sw_alb",
-) where {FT}
+) where {FT <: AbstractFloat}
```

Comment:
do we need this? or is FT already defined to be a subtype of AbstractFloat somewhere else (defn of BulkAlbedoTemporal?)